app-k9mail/src/main/kotlin/com/fsck/k9/K9App.kt

```diff
@@ -0,0 +1,7 @@
+package com.fsck.k9
```

Comment:
We could use this opportunity to change the package name to `app.k9mail`.

Reply:
I would to that in a separate pull-request